### ISSUE TITLE
FreeSurfer Normalized Image

### ISSUE THREAD

**Issue by oberoigaurav** [voxelmorph/voxelmorph#147]
**Task** (Checking the performance of trained Voxelmorph on my data.)

On doing so, I found that Voxelmorph is not giving results for my data. Then I found that the given Atlas and test image has intensity values between 0 and 1 and my FreeSufer normalized images have intensity values ranging from 0 to 110. 

Which library/method you have used for Intensity Normalization between 0 and 1??

**Comment by adalca** [comment]
If you have freesurfer-processed images (norm.mgz), we just divided by 255.

**Comment by oberoigaurav** [comment]
Okay, thanks !!

**Comment by oberoigaurav** [comment]
Hi,
Finetuning or training your model on my dataset is degrating the performance of the model. I am using the hyperparameters provided by you, i.e., Loss=ncc, LR=1e-4, Regularization=1. Epochs=1500, steps_per_epoch=100. I am not sure why it is happening.

Before Fine Tuning, NCC on my dataset = 92%
After Fine Tuning, NCC on my dataset = 78 %

Are the parameters above optimal ? 
Thanks in advance !

**Comment by adalca** [comment]
Can you please provide information about your dataset?

Also, what metric are you using to get percentage-based errors?

**Comment by oberoigaurav** [comment]
I am using Mindboggle-101 dataset https://www.frontiersin.org/articles/10.3389/fnins.2012.00171/full
I have normalized and brain extracted the dataset using FreeSurfer recon-all pipeline.
Then I have cropped the dataset to size 160X192X224 and divided by 255 to get intensity values between 0 and 1. 


I am using Normalized Cross-correlation(NCC) as the similarity metric. (Multipying by 100 to get percentage NCC)

**Comment by adalca** [comment]
NCC is not a good measure for accuracy, since if all you wanted was NCC to be high you then you wouldn't regularize. See [CURT](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3274625/) for a discussion of this. To know if your results are good, you should use something like Dice with segmentations, or TRE. Or visualize: 

`moving | fixed | moved | deformation_field`

like in our papers.
